src/services/filesSharingServices.js

```diff
@@ -55,6 +55,25 @@ const shareFile = async function(path, token, referenceId, metadata) {
 	}
 }
 
+/**
+ * Share a text file to a conversation
+ *
+ * @param { string } filePath the file path
+ * @return { object } the file object
+ */
+const createTextFile = async function(filePath) {
+	try {
+		return axios.post(
+			generateOcsUrl('apps/files/api/v1/templates/create'),
+			{
+				filePath,
+			})
+	} catch (error) {
+		console.debug(error)
```

Comment:
When a file with the same name already exists, this fails silently and you don't see anything.
Your button press just seems to be ignored.